extensions/amp-consent/0.1/cmps.js

```diff
@@ -33,4 +33,10 @@ if (getMode().test || getMode().localDev) {
     'checkConsentHref': '/get-consent-v1',
     'promptUISrc': '/test/manual/diy-consent.html',
   };
+
+  CMP_CONFIG['didomi'] = {
+    'consentInstanceId': 'didomi',
+    'checkConsentHref': 'https://api.privacy-center.org/amp/check-consent',
+    'promptUISrc': 'https://sdk-amp.privacy-center.org/loader.html',
+  };
 }
```

Comment:
Also, I just noticed, you wrapped your CMP in the `if (getMode().test || getMode().localDev) {`. 

I suspect you want your CMP to be available outside of testing and development mode correct? As in, you want PROD users to be able to use your CMP? Could you move this out of the `if` block?

Thanks! 😄

Reply:
@torch2424 sure, you're right. I'll do the changes. 

Thank you :)